src/docparser.cpp

```diff
@@ -963,6 +963,10 @@ void DocParser::defaultHandleTitleAndSize(const int cmd, DocNodeVariant *parent,
       // special case: no title, but we do have a size indicator
       break;
     }
+    else if (tok==TK_HTMLTAG)
+    {
+      break;
```

Comment:
should we no unput the string like is done below? For a closing tag is it probably ok to ignore it, but an opening tag should not be ignored, like in the following example:

```html
<table border=0>
<tr>
<td>\image html sphere.png<tr align="center">
<td>(a) Sphere</td>
</tr>
</table>
```

Reply:
Correct, none of the tags should be ignored. I looked like it was working after the warning was gone but this was luck.

Code was inspired by the "height=/width=" above, but that is a different situation where later on the part after the part after the `=` is processed and there there is no unput necessary. 

The pull request has been updated.